test/_accessibility/accept/accessibility.spec.js

```diff
@@ -30,14 +32,26 @@ describe('the journey of an accessible accept application', async () => {
     getUrl = testApp.getUrl;
   });
 
+  async function content(pathValue) {
+    try{
+      const htmlText =  await readFile(pathValue, 'utf8');
+      return htmlText;
+    } catch (err) {
+      console.log(err);
+      return err;
+    }
+  }
+
   it('check accept accessibility issues', async () => {
+    let browser;
     await initSession(URI);
 
     const exclusions = [
       '/confirm',
       '/complete-acceptance'
     ];
 
+    console.log('uris: ', uris);
```

Comment:
Best to remove all console.logs before merging

Reply:
I have added these console log to group pa11y result and page information together and it will help to debug if any pa11y result show any issue